### ISSUE TITLE
[Feature Suggestion] Manually Set Cache and Settings location

### ISSUE THREAD

**Issue by mattbarreiro** [qarmin/czkawka#880]
It would be great to be able to override the default cache and settings locations, and allow setting custom values for each. Or at least the ability to store the cache and settings in the same folder as the binary. 

Use case: 
Store cache and settings on a network share. Allows me to swap between different computers when de-duping files. Also I could use my desktop to pre-calculate all the hashes, and then work on actually de-duping from a laptop on my couch 😂 (without needing to leave both on/deal with RDP)

Ideas: : 
* Have the program check if `settings_*` or `cache_*` exists in the same folder as the binary. If yes, load those instead of loading from default locations. 
* Put some sort of override file in the same directory as the binary (e.g.`settings_override.txt` that contains the path). 
* Utilize ENVs instead of override files

Thank you for all of your work on this project!

**Comment by PhuriousGeorge** [comment]
+1 on this idea with the addition of the ability to change default required file size for cache.

I deal with millions of images just under the 5MB required file size for cache and it would be extremely useful not to have to scan my network shares every time I want to ensure I'm not adding duplicates to my collection.

**Comment by Le-zOU** [comment]
I agree, the ability of setting a manual cache location would be essential as default location in windows depends on OS partition size

which can be quite limited

C:\Users\*****\AppData\Local\Qarmin\Czkawka\cache

**Comment by qarmin** [comment]
This was already added in previous versions(can be changed, by changing envs)
